src/main/scala/jigg/util/JSONUtil.scala

```diff
@@ -116,6 +120,64 @@ object JSONUtil {
   }
 
   def writeToJSON[T <: Node](x: T, os: BufferedWriter): Unit = {
-    os.write(toJSON(x).toString)
+    os.write(toJSON(x))
+  }
+
+  def parseJSON(json: String): JValue = parse(json)
+
+  def loadJSONFile(name: String): JValue = {
+    val sb = new StringBuilder
+    for (line <- Source.fromFile(name).getLines){
+      sb.append(line)
+    }
+    parse(sb.toString)
+  }
+
+  def toXML(json: JValue): Node = {
+    implicit val formats = DefaultFormats
+    val jsonList = json.extract[Map[String, Any]].toList
+    generateXML(jsonList)
+  }
+
+  private def generateXML(x:List[(String, Any)]): Node = {
+    var tagString = ""
+    var textString = ""
+    val attributeList = new ArrayBuffer[(String, String)]
+    val nodeSeq = new ArrayBuffer[Node]
+    var hasChildFlag = false
+    var hasTextFlag = false
+
+    for (i <- x){
```

Comment:
Thank you very much for your comment and code.
I'm going to use your code as a reference.